docs/release-notes_exporter_v2.0.0-rc.1.md

```diff
@@ -0,0 +1,255 @@
+- `errors?: string[]`.
+
+**Examples**
+```bash
+curl -s "http://localhost:16000/v1/exporter/traces/committee?from=123456&to=123460&committeeIDs=<64-hex>"
+
+curl -s -X POST http://localhost:16000/v1/exporter/traces/committee   -H 'Content-Type: application/json'   -d '{"from":123456,"to":123460,"committeeIDs":["<64-hex>"]}'
+```
+
+---
+
+### `GET|POST /v1/exporter/decideds`  _(both modes)_
+
+- **Standard mode:** returns the *v1 payload*: `{ data: ParticipantResponse[] }`.  
+- **Archive mode:** same path and returns `{ data: DecidedParticipant[], errors?: string[] }`. The `errors` field is **additive** and does not remove existing fields (`role`, `slot`, `public_key`, `message.Signers`).
+
+**Filters** (`DecidedsRequest`)
+- `from`, `to` — inclusive slot range (int64 ≥ 0).
+- `roles` — same enum as above.
+- `pubkeys` — 96‑hex pubkeys.
+- `indices` — validator indices (int64 ≥ 0).
+
+**Behavior (best‑effort)**  
+Expected “not found” conditions (e.g., no duty for a slot) are **not** treated as API errors. Non‑fatal processing issues are aggregated into `errors`. If **no valid results** are found **and** at least one meaningful error exists, the API returns an error; otherwise it returns partial data plus `errors`.
+
+Note: In standard mode, the `indices` filter is ignored for this endpoint; use `pubkeys` to filter. In archive mode, both `pubkeys` and `indices` are supported.
+
+**Examples**
+```bash
+# Standard mode → v1 payload
+curl -s "http://localhost:16000/v1/exporter/decideds?from=123456&to=123460&roles=ATTESTER&pubkeys=<96-hex>"
+
+# Archive mode → richer payload with `errors`
+curl -s "http://localhost:16000/v1/exporter/decideds?from=123456&to=123460&roles=ATTESTER"
+```
+
+---
+
+## WebSocket API
+
+When `WebSocketAPIPort` is configured and the exporter is enabled:
+
+- `ws://<host>:<port>/stream` — **decided (quorum) events** published once per `(validator, role, root)` with a **1‑minute de‑duplication window**.  
+  Where possible, listeners backfill pubkeys from the registry for retro‑compatibility.
+- `ws://<host>:<port>/query` — retained for ad‑hoc lookups (internal format), unchanged from v1.
+
+---
+
+## Compatibility & Behavior Changes
+
+- **Retro‑compatibility:** The legacy `/v1/exporter/decideds` remains available in both modes. In archive mode it is powered by the new backend and **adds** `errors` to the response (existing fields preserved). Clients that strictly validate schemas should allow unknown fields.
+- **Two modes:** Archive (full traces, Pebble, all subnets) vs. Standard (participants only, pruning).
+- **Best‑effort processing:** Partial results with `errors` instead of hard failures where possible; empty‑with‑errors yields HTTP error.
+- **Rate limiting:** APIs may respond with `429 Too Many Requests`; clients must implement backoff with jitter.
+- **OpenAPI:** Specs are versioned under `docs/api/ssvnode.openapi.yaml|json`.
+
```

Comment:
seems like all of these been mentioned before already.. if anything new here lets take it out and concentrate all in one section.